models/cam-fv/readme.rst

```diff
@@ -1,208 +1,314 @@
+It contains just under 120Tb (yes Tb) of data:
 
-   These CAM6 Data Assimilation Research Testbed (DART) Reanalysis data 
+   These CAM6+DART Reanalysis data 
    products are designed to facilitate a broad variety of research using 
    NCAR's CESM2 models, ranging from model evaluation to (ensemble) 
-   hindcasting, data assimilation experiments, and sensitivity studies. 
+   hindcasting (initial conditions), data assimilation experiments, and sensitivity studies. 
    They come from an 80 member ensemble reanalysis of the global 
-   troposphere and stratosphere using DART and CAM6 from CESM2.1. 
+   troposphere and stratosphere using CAM6-FV from CESM2.1. 
    The data products represent the actual states of the atmosphere 
-   during 2 recent decades at a 1 degree horizontal resolution and 
+   from 2011-2019 at a ~1 degree horizontal resolution and up to 
    6 hourly frequency. Each ensemble member is an equally likely 
    description of the atmosphere, and is also consistent with 
-   dynamics and physics of CAM6.
+   dynamics and physics of CAM6-FV.
    
    
-The NCAR Research Archive dataset **"An Ensemble of Atmospheric Forcing Files from a CAM Reanalysis"**
-`DS199.1 <https://rda.ucar.edu/datasets/ds199.1/#!description>`__ contains
-about 1.5Tb of data:
+An earlier, more limited dataset can be found in the 
+`**Ensemble of Atmospheric Forcing Files from a CAM4-FV Reanalysis**
+   <https://github.com/NCAR/DART/wiki/2-degree-DATM-ensemble-for-CESM-experiments-(1998-thru-2010)>`__
+is archived in the NCAR Research Data Archive 
+`DS199.1 <https://rda.ucar.edu/datasets/ds199.1/#!description>`__ .
+It contains about 1.5Tb of data:
 
    This dataset contains files that are an ensemble of 'coupler history' 
    files from an 80-member reanalysis performed with the Data Assimilation 
    Research Testbed (DART) using the Community Atmosphere Model Version 
-   4 with the finite volume core (CAM4 FV) at 1.9 degree by 2.5 degree 
+   4 with the finite volume core (CAM4-FV) at 1.9 degree by 2.5 degree 
    resolution. The observations assimilated include all those used in 
    the NCEP/NCAR reanalysis (temperature and wind components from 
    radiosondes, aircraft, and satellite drift winds) plus radio 
    occultation observations from the COSMIC satellites starting in late 
    2006. These files are intended to be used as 'DATM stream files' 
    for CESM component sets that require a data atmosphere. Some example 
-   stream text files are included to illustrate how to use these data.
-
-**Guidance**
-
-Experience on a variety of machines has shown that it is a very good idea to
-make sure your run-time environment has the following:
-
-.. code-block:: bash
+   stream text files are included in the RDA to illustrate how to use these data.
+
+..  * CAM4, 2 degree, 2000-2010, `Reanalysis <https://rda.ucar.edu/datasets/ds199.1>`__
+..  * files from the old "large file site": http://www.image.ucar.edu/pub/DART/CAM/
```

Comment:
These are .. comments.  

Should these lines be in the documentation?

Reply:
For the moment I thought they should be in the documentation of the documentation.
I don't know whether or how we'll use the "large file site", so I was keeping a reminder here.
If these .. comments will be harvested and displayed, then this one shouldn't be here.
And others.